sources/lib/api/gcp/stackdriver/monitoring/_impl/api.py

```diff
@@ -0,0 +1,283 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""HTTP API wrapper for the Google Monitoring API."""
+
+try:
+  from gcp._util import Http, RequestException
+except ImportError:
+  # Use a minimal substitute module if the Datalab library is not available.
+  from .http import Http, RequestException
```

Comment:
Under what circumstances do you think the Datalab library would not be available?